### ISSUE TITLE
Add configurable grace period for detecting inputs

### ISSUE THREAD

**Issue by alice-i-cecile** [Leafwing-Studios/leafwing-input-manager#48]
Chords detected within this grace period should be treated as one unit.

**Comment by plof27** [comment]
This should be changed to be more mindful of the accessibility use case, as that is the main value in this enhancement IMO. Focusing on the accessibility side, it would look more like a configurable "sticky keys" setting for chorded inputs.

**Comment by alice-i-cecile** [comment]
This should also work with arpeggios, by allowing momentary gaps between the inputs.
